.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Dragonfly GDB",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/build-dbg/dragonfly",
```

Comment:
Nice! Is it possible somehow to work on multiple targets, i.e. unit tests and dragonfly?

Reply:
So actually if we allow vscode to build by itself, it automatically knows how to debug all targets (without this file)!
I struggled with it a bit but couldn't get it to work like `blaze.sh` so gave up for now